src/molecules/BundleDealBox/BundleDealBox.pcss

```
@@ -0,0 +1,79 @@
+.bundleDealBox {
```

```suggestion
.telia-bundle-deal-box {
```
We should prefix new components with `telia` for namespacing.